great_expectations/data_context/data_context.py

```diff
@@ -296,14 +296,19 @@ def __init__(self, project_config, context_root_dir=None, runtime_environment=No
         # the way that execution environments (AKA datasources), validation operators, site builders and other
         # plugins are built.
         self.validation_operators = {}
-        for (
-            validation_operator_name,
-            validation_operator_config,
-        ) in self._project_config.validation_operators.items():
-            self.add_validation_operator(
+        # NOTE - 20210112 - Alex Sherstinsky - Validation Operators are planned to be deprecated.
+        if (
```

Comment:
@roblim Good question (because this hit a small bug during development): We check that the key is there AND that the value has something meaningful in it.  So one has to have both.

Reply:
Oh, gotcha. Makes sense now!